go.mod

```diff
@@ -1,3 +1,16 @@
-module github.com/turnage/graw
+module github.com/monkeydioude/graw
```

Comment:
This should not be changed.

Reply:
Will do, needed it for testing the package